samples/modules/lvgl/demos/boards/mimxrt595_evk_mimxrt595s_cm33.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright 2024, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Sample will allocate buffer equal to: (panelwidth / 8) * (panelwidth / 4) * pixel depth. For a
+# 1280x720 display in a 16-bpp format (e.g. RGB565), this is (1280 / 8) * (720 / 4) * 2 = 57600
+# bytes. We include 128 bytes of padding for kernel heap structures
```

Comment:
Is this still correct?

Reply:
according to the dts for this shield, it is RGB565@1280x720

Reply:
@faxe1008 I have update above from RGB888 to RGB565, please check. Thanks